bundles/org.openhab.binding.panasonicbr/README.md

```diff
@@ -0,0 +1,242 @@
+# Panasonic Blu-ray Player Binding
+
+This binding connects Panasonic Blu-ray players from 2011/2012 and UHD Blu-ray players from 2018 to openHAB.  
+**Supported Blu-ray models:** DMP-BDT110, DMP-BDT210, DMP-BDT310, DMP-BDT120, DMP-BDT220, DMP-BDT320, DMP-BBT01 & DMP-BDT500.  
+**Supported UHD models:** DP-UB420/424, DP-UB820/824 & DP-UB9000/9004.  
+
+**Please Note:** The player must be on the same IP subnet as the openHAB server for this binding to function.
+If the connection to the player originates from a different subnet, 404 response errors are sent in response to all requests.
+
+To enable network remote control of the Blu-ray model players, configure the following settings:  
+**Player Settings/Network/Network Settings/Remote Device Settings**  
+Then make sure you have the following values set:  
+**Remote Device Operation: On**  
+**Registration Type: Automatic**  
+
+For the UHD models, Voice Control must be enabled for the player's http interface to be active:
+**Player Settings/Network/Voice Control: On**
+
+To enable the binding to control the player while off (network active while off), Quick Start mode must be On as follows:  
+**Player Settings/System/Quick Start: On**  
+
+**UHD Model Command Authentication:**  
+The UHD models require authentication to use the control API.
+A player key must be specified in the thing configuration in order for the `power`, `button` and `control` channels to work.
+UHD model players that are patched do not require a player key.
+See the online discussions of the DP-UB420/820/9000 players for more information.
+
+## Supported Things
+
+There are two supported thing types, which represent either a BD player or a UHD player.
+A supported Blu-ray player uses the `bd_player` id and a supported UHD Blu-ray player uses the `uhd_player` id.
+Multiple Things can be added if more than one player is to be controlled.
+
+## Discovery
+
+Auto-discovery is supported if the player can be located on the local network using UPNP.
+Otherwise the thing must be manually added.
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+| Parameter | Description                                                                                           |
+|-----------|-------------------------------------------------------------------------------------------------------|
+| hostName  | The host name or IP address of the player. Mandatory.                                                 |
```

Comment:
For your consideration, add columns if the config parameter is required and/or if it has a default